alchemist-graphql/src/jvmMain/kotlin/it/unibo/alchemist/boundary/graphql/schema/model/surrogates/TimeSurrogate.kt

```diff
@@ -9,14 +9,20 @@
 
 package it.unibo.alchemist.boundary.graphql.schema.model.surrogates
 
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
 import it.unibo.alchemist.model.Time
 
 /**
  * A GraphQL surrogate for a [Time].
  */
 enum class TimeSurrogate {
```

Comment:
How can this work? The time is usually a number

Reply:
I have misinterpreted the `Time` class. I have already added the correct implementation.